apps/prairielearn/src/ee/pages/instructorAiGenerateQuestion/instructorAiGenerateQuestion.html.ts

```diff
@@ -36,109 +39,337 @@ const examplePrompts = [
-                    </em>
-                  </div>
-                </div>
-                <div class="form-group">
-                  <label for="user-prompt-llm-user-input">
-                    How should students input their solution? What choices or input boxes are they
-                    given?
-                  </label>
-                  <textarea
-                    name="prompt_user_input"
-                    id="user-prompt-llm-user-input"
-                    class="form-control"
-                  ></textarea>
-                  <div class="form-text form-muted">
-                    <em>
-                      Example: students should enter the solution using a decimal number. The answer
-                      should be in seconds.
-                    </em>
-                  </div>
-                </div>
-                <div class="form-group">
-                  <label for="user-prompt-llm-grading">
-                    How do we determine if a solution is correct/which answer choice is correct?
-                  </label>
-                  <textarea name="prompt_grading" id="user-prompt-llm-grading" class="form-control">
-                  </textarea>
-                  <div class="form-text form-muted">
-                    <em>
-                      Example: the answer is computed as sqrt(2 * h / g) where g = 9.81 m/s^2
-                    </em>
-                  </div>
-                </div>
-                <button class="btn btn-primary">
-                  <span
-                    class="spinner-grow spinner-grow-sm d-none"
-                    role="status"
-                    aria-hidden="true"
-                    data-loading-class-remove="d-none"
-                  >
-                  </span>
-                  Create question
-                </button>
-              </form>
-              Or choose a question from our list of example prompts:
-              <select
-                id="user-prompt-example"
-                onchange="setPromptToExample()"
-                class="custom-select"
-              >
-                <option value=""></option>
-                ${examplePrompts.map(
-                  (question) =>
-                    html`<option
-                      value="${question.id}"
-                      data-prompt-general="${question.promptGeneral}"
-                      data-prompt-user-input="${question.promptUserInput}"
-                      data-prompt-grading="${question.promptGrading}"
-                    >
-                      ${question.id}
-                    </option>`,
-                )}
-              </select>
-              <div id="generation-results"></div>
-              <br />
-              <div>
-                <a href="${resLocals.urlPrefix}/ai_generate_question_jobs" class="btn btn-primary">
-                  Previous Generation Jobs
-                </a>
+              ${
+                !prompts || prompts.length <= 0
```

Comment:
if qid is null or there's no recorded prompts for the qid, then we're in question creator [fallback] mode (see comment below)

Reply:
This is no longer relevant after splitting the pages.